FileProcessor/src/main/java/elements/Filter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package elements;
+
+import com.group7.FileProcessor.pojo.EntriesPOJO;
+import java.util.ArrayList;
+
+/**
+ *
+ * @author keric
+ */
+public abstract class Filter {
+
+    private ArrayList<EntriesPOJO> inputEntries;
```

Comment:
This looks really good. Is it possible though it would be better if this were to use the Entry class Shalev is working on? Instead of the POJO?

Reply:
quite possibly. It's easy to change.

Reply:
okay I will try and get him to finish that work and get it into main as fast as possible